examples/python/plots/plots.py

```diff
@@ -43,9 +43,6 @@ def log_bar_chart() -> None:
 
 
 def log_parabola() -> None:
-    # Name never changes, log it only once.
-    rr.log("curves/parabola", rr.SeriesLine(name="f(t) = (0.01t - 3)³ + 1"), static=True)
-
```

Comment:
I think it would be good to show the old API as an alternative in one of these plots (with a comment)

Reply:
done